### PR TITLE
clean up integration

### PR DESCRIPTION
**Pull request by tjhei**

### CODE REVIEW

File: biharmonic.cc [hunk 1]
```diff
@@ -529,26 +524,24 @@ namespace StepBiharmonic
...
+		    //
+                    + 2.0 * gamma * (fe_i.jump_gradient(i, qpoint) * n) // 2 gamma [grad v n]
+		    * (fe_i.jump_gradient(j, qpoint) * n)) // [grad u n]
+		  * JxW[qpoint]; // dx
 
 
               copy_data.cell_rhs(i) +=
-                (-(fe_i.average_hessian(i, qpoint) * n * n) *
-                   (exact_gradients[qpoint] * n) //
-                 + 2.0 * gamma *                 // why 2?
```
Comment (bangerth) [inline biharmonic.cc:547]: so you found that the factor of 2 is correct?

**Review by bangerth — COMMENTED** [review]
So do I see it right that the implementation was correct and that all you do here is add some commentary and move symbols around?

**Comment by bangerth** [comment]
Turns out that with your use of the mesh loop framework + `FEInterfaceValues` and my cleaning up of the code, it's actually quite a neat little code that's not all that hard to understand :-)

**Comment by tjhei** [comment]
> So do I see it right that the implementation was correct and that all you do here is add some commentary and move symbols around?

this is just rearranging and commenting.

> so you found that the factor of 2 is correct?

I don't think it matters, one could put it inside gamma as well. Note that the parameter design for gamma is still quite ugly/unfinished/etc..

> it's actually quite a neat little code that's not all that hard to understand :-)

yep :)

**Pull request merged**